### DISCUSSION TITLE
Duplcate text styles since last update.

### DISCUSSION THREAD

**Discussion by bmooij-lg** [ingrammicro/puzzle-tokens#53] [Q&A]
Since the fix in https://github.com/ingrammicro/puzzle-tokens/discussions/52 all text styles are added again, as opposed to updated. I've tried enabling the setting you suggested in that discussion, but that didn't fix it.

Sidenote: again, this doesn't seem to happen when running the plugin directly from sketch, only when calling it up by code...

**Comment by MaxBazarov** [comment]
Ok… reviewing

**Comment by MaxBazarov** [comment]
Fixed  in 8.19.2
Sorry for the issue

**Reply by bmooij-lg** [reply]
It could be my setup, but it's no longer applying the different styles.
It now seems to run the .less file last chosen manually. Regardless of the file/path defined
